### PR TITLE
Add fixes for SR-IOV VF partitions

### PR DESCRIPTION
**Pull request by jboxman**
Minor fixes.

### CODE REVIEW

**Comment by jboxman** [comment]
@zshi-redhat, I made some minor formatting changes; let me know if this looks okay. Thanks!

File: modules/nw-sriov-nic-partitioning.adoc [hunk 1]
```diff
@@ -20,10 +22,10 @@ pfNames: ["netpf0#2-7"]
 
 You can select VFs from the same PF by using different policy CRs if the following requirements are met:
 
-* The `numVfs` must be identical for policies that select the same PF.
-* The VF index must be in range of `0` to `<numVfs>-1`. For example, if you have a policy with `numVfs` set to `8`, then the `<FirstVF>` must not be smaller than 0, and the `<LastVF>` must not larger than 7.
+* The `numVfs` value must be identical for policies that select the same PF.
+* The VF index must be in range of `0` to `<numVfs>-1`. For example, if you have a policy with `numVfs` set to `8`, then the `<first_vf>` must not be smaller than 0, and the `<last_vf>` must not larger than 7.
 * The VFs ranges in different policies must not overlap.
-* The `<FirstVF>` must not larger be than the `<LastVF>`.
+* The `<first_vf>` must not larger be than the `<last_vf>`.
```
Comment (zshi-redhat) [inline modules/nw-sriov-nic-partitioning.adoc:28]: nit:  `larger be` => `be larger`.

**Review by zenghui-shi — COMMENTED** [review]
@jboxman this change looks good to me. just one nit.

**Pull request merged**